features/site-empty.feature

```diff
@@ -42,6 +42,12 @@ Feature: Empty a WordPress site of its data
     When I run `wp term list post_tag --format=ids`
     Then STDOUT should be empty
 
+    When I run `wp option get wp_page_for_privacy_policy`
+    Then STDOUT should be:
+      """
+      0
+      """
```

Comment:
Can we also include a test that makes sure there's a non-zero value _before_ `wp site empty` is run?

Reply:
I am not sure how to do that, should I create a dummy page and assign it as privacy page?

Reply:
> should I create a dummy page and assign it as privacy page?

Yes, this would be fine.